include/wx/msw/bitmap.h

```diff
@@ -199,8 +199,11 @@ class WXDLLIMPEXP_CORE wxBitmap : public wxGDIImage,
     void MSWUpdateAlpha();
 
 public:
-    void SetHBITMAP(WXHBITMAP bmp) { SetHandle((WXHANDLE)bmp); }
+#if WXWIN_COMPATIBILITY_3_0
+    wxDEPRECATED_INLINE(void SetHBITMAP(WXHBITMAP bmp), SetHandle((WXHANDLE)bmp); )
+#endif // WXWIN_COMPATIBILITY_3_0
     WXHBITMAP GetHBITMAP() const { return (WXHBITMAP)GetHandle(); }
+    void SetHBITMAP(WXHBITMAP bmp, int width, int height, int depth);
```

Comment:
This is minor, but I think it might be better to call this function `InitHBITMAP()` (or maybe `InitFromHBITMAP()`?) rather than `Set` for 2 reasons: first, to avoid any confusion with the deprecated overload and second, to indicate that it can/should be used with the yet otherwise non-initialized bitmap.